app/controllers/CalculationController.scala

```diff
@@ -60,8 +61,8 @@ class CalculationController @Inject()(implicit val config: FrontendAppConfig,
             calculationService.getFinancialData(user.nino, taxYear).map {
               case calcDisplayModel: CalcDisplayModel =>
                 submitData(user, sources, calcDisplayModel.calcAmount.toString)
-                if(calcDisplayModel.crystalisedFlag.equals("Crystalised")){
-                  Ok(views.html.crystalised(calcDisplayModel, taxYear))
+                if(calcDisplayModel.crystallisedFlag.equals(Crystallised)){
```

Comment:
Can this be called `calcStatus` rather than `crystallisedFlag` as this could cater for more than just crystalised calculations in the future. Also, can this be changed to a pattern match for the same reason. I.e.:

```
calcDisplayModel.calcStatus match {
   case calc: Crystallised => Ok(views.html.crystallised(calc, taxYear))
   case calc: Estimate => Ok(view.htmls.estimatedTaxLiability(calc, taxYear))
}
```